src/chunktransformer.js

```diff
@@ -87,8 +87,8 @@ ChunkTransformer.prototype._transform = function(data, enc, next) {
       return;
     }
   }
-  if (data.lastScannedRowKey) {
-    this.lastRowKey = data.lastScannedRowKey;
+  if (data.lastScannedRowKey && data.lastScannedRowKey.length > 0) {
```

Comment:
[lastScannedRowKey is defined as bytes](https://github.com/googleapis/nodejs-bigtable/blob/8db7bf35f79de791dd5adbad55625048c79694ad/protos/google/bigtable/v2/bigtable.proto#L181) so it will always be set to at least an empty array even when the service didn't actually set it